README.md

```diff
@@ -1,27 +1,21 @@
+[![Build Status](https://travis-ci.com/HumanCompatibleAI/airl.svg?branch=master)](https://travis-ci.com/HumanCompatibleAI/airl)
+
 # Adversarial Inverse Reinforcment Learning
 
 This repository reimplements AIRL with modern dependencies.
 Paper: https://arxiv.org/abs/1710.11248
 Original Implementation: https://github.com/justinjfu/inverse_rl
 
 To install:
-
 ```
-conda create -n yairl python=3.6.8  # Tensorflow doesn't work with python 3.7 at the time of writing.
+sudo apt install libopenmpi-dev
+conda create -n yairl python=3.7.2
```

Comment:
If it works with the latest version of python, just leave off the 'python=3.7.2' so conda can install the most up to date one.